.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,28 +1,28 @@
-<!-- Hey there. Thank you so much for improving NewPipe. Please take a moment to fill out the following suggestion on how to structure this PR description. Having roughly the same layout helps everyone considerably :)-->
+<!-- Hey there. Thank you so much for improving NewPipe, and filling out the details. Having roughly the same layout helps everyone considerably :)-->
 
 #### What is it?
-- [ ] Bug fix (user facing)
+- [ ] Bugfix (user facing)
 - [ ] Feature (user facing)
-- [ ] Code base improvement (dev facing)
+- [ ] Codebase improvement (dev facing)
 - [ ] Meta improvement to the project (dev facing)
 
 #### Description of the changes in your PR
-<!-- While bullet points are the norm in this section, feel free to write a text instead if you can't fit it in a list -->
+<!-- While bullet points are the norm in this section, feel free to write a text instead of a list -->
 - record videos
 - create clones
 - take over the world
 
 #### Fixes the following issue(s)
-<!-- Also add reddit or other links which are relevant to your change. -->
+<!-- Also add non-reddit or other links relevant to your change. -->
```

Comment:
Why non-reddit? reddit is explicitly allowed.

Reply:
That would be Reddit. I mean "reddit" as in the verb, meaning questionable content.

Reply:
Ah. ok :D We meant Reddit.

Reply:
Yes, but saying it like that would be entrapment. :)

Reply:
> I mean "reddit" as in the verb, meaning questionable content.

I don't think 'reddit' is a word in English at all.